### DISCUSSION TITLE
Curso Fullstack .Net (Testes)

### DISCUSSION THREAD

**Discussion by FBODeveloper** [balta-io/forum#662] [Tira Dúvidas] [answered]
Pessoal, sou leigo em testes...
No curso Fullstack se for criar testes devo testar os models ou request? Ou ambos?

Valeu!!!!

**Comment by andrebaltieri** [comment, marked as answer]
@FBODeveloper os Handlers! Como criamos interfaces, você pode mockar a maioria das coisas e testá-las.
No nosso caso, os Models e Requests não tem regras de negócio.
